.github/label-actions.yml

```diff
@@ -3,10 +3,14 @@ Needs Logs:
   comment: "We need more info to debug your particular issue. If you could attach your logs to the issue (ensure no private data is in them), it would help us fix the issue much faster.
```

Comment:
You need to use the pipe operator - see here for an example : 

https://github.com/Azure/azure-functions-sql-extension/blob/main/.github/label-actions.yml#L3

https://github.com/Azure/azure-functions-sql-extension/pull/719

(it's called a "literal scalar block" - see https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines for an explanation of the horribly-confusing amount of ways to do multi-line content)